build.xml

```diff
@@ -1031,7 +1031,8 @@
 
         <source path="${basedir}/src/java/" />
         <source path="${basedir}/src/test/" output="build/test/classes" />
-
+        <source path="${plugins.dir}/any23/src/java/" />
+        <source path="${plugins.dir}/any23/src/test/" />
```

Comment:
Please add entries for the ```javadoc``` and ```eclipse``` targets as well. 

Additionally, please add entries to ```default.properties``` as appropriate

Reply:
a1bcad5